frontend/src/pages/user/seller/allReports/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+
+import TButton from '@components/TButton';
+import FormItem from '@components/FormItem';
+
+const SellerReport = () => {
+  const now = new Date();
+  const [year, setYear] = useState(now.getFullYear());
+  const [month, setMonth] = useState(now.getMonth());
+  return (
+    <div style={{ padding: '5% 8% 10% 8% ' }}>
+      <h3>At what time would you like to view the report?</h3>
```

Comment:
this text is not very "user-friend".
I think using "Select Date" or "Report Period".